homeassistant/components/websocket_api/connection.py

```diff
@@ -157,10 +158,11 @@ def async_handle(self, msg: dict[str, Any]) -> None:
                 or type(type_) is not str  # pylint: disable=unidiomatic-typecheck
             )
         ):
-            self.logger.error("Received invalid command", msg)
+            self.logger.error("Received invalid command: %s", msg)
+            id_ = msg.get("id") if isinstance(msg, dict) else 0
```

Comment:
Not use this would work too?

```suggestion
            id_: int = msg.get("id") if isinstance(msg, dict) else 0
```

Reply:
Thanks, Let's give it a shot and see what the ci thinks.  At this point it Gigo anyways and it's only for devs to try to get a better error